### PR TITLE
Tell contained resources what container saves them

### PR DESCRIPTION
**Pull request by tpendragon**

### CODE REVIEW

**Comment by mjgiarlo** [comment]
:ok_hand:

File: spec/integration/contains_association_spec.rb [hunk 1]
```diff
@@ -16,7 +16,9 @@ class Source < ActiveFedora::Base
       s = Source.new
       s.sub_resource.title = "Test"
       expect(s.sub_resource).not_to be_persisted
+      allow(s.sub_resource).to receive(:save).and_call_original
       expect { s.save }.not_to raise_error
+      expect(s.sub_resource).to have_received(:save).with(container: s)
```
Comment (jcoyne) [inline spec/integration/contains_association_spec.rb]: I'm not sure we should be using spys in an integration test. Can you write a model that would actually use the new behavior?

Comment (tpendragon) [inline spec/integration/contains_association_spec.rb]: Sure, there ya go.

**Comment by jcoyne** [comment]
:+1:

**Comment by tpendragon** [comment]
This isn't useful to the AF::Aggregations PR anymore, so closing it.

**Pull request closed without merging**